test/parallel/test-runner-run.mjs

```diff
@@ -488,39 +502,6 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     });
   });
 
-  it('should run with no files', async () => {
```

Comment:
Should these tests be removed? Shouldn't these still work the same?

Reply:
Unfortunately they will now fail as they are trying to load _all_ files in `test` that matches the given glob pattern. They were passing before due to the CLI checks that I removed.